src/print.jl

```diff
@@ -201,7 +201,7 @@ end
 function model_string(print_mode, model::AbstractModel)
     ijl = print_mode == IJuliaMode
     sep = ijl ? " & " : " "
-    eol = ijl ? "\\\\\n" : "\n"
+    eol = ijl ? "\\\\" : "\n"
```

Comment:
Is this due to a bug in Documenter that is reported somewhere or a known limitation ?

Reply:
It was the same error as this: https://github.com/JuliaDocs/Documenter.jl/issues/1387.

Looking through your PR, I guess the regex needs a suffix `"s` so it can match multiline strings:
https://github.com/JuliaDocs/Documenter.jl/pull/1426/files#diff-c34d581afcb6837fd39a6b0b0428090c049d7dad13de417d96dc08a0ed01cf70R1751-R1752

Reply:
https://github.com/JuliaDocs/Documenter.jl/pull/1518

Reply:
So the newline issue can be fixed upstream, but there is still the issue that we are using `alignat` instead of `aligned`. With this PR, the tutorials print nicely.